src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -450,14 +401,12 @@ private Error buildErrorWithParameter(String value, ErrorCodes errorCode) {
       .withParameters(parameters);
   }
 
-  private String buildProductIdQuery(ProductId productId, Map<String, String> productTypes) {
-    return String.format("(identifiers adj \"\\\"identifierTypeId\\\": \\\"%s\\\"\" " +
```

Comment:
Please return this part of the query as dentifierTypeId is important for finding an instance.

Reply:
Now it shoul be `String.format("...", productId.getProductIdType(), productId.getProductId())`

Reply:
Done

Reply:
Done